client/packages/common/src/ui/layout/tables/components/Cells/NumberInputCell/NumberInputCell.tsx

```diff
@@ -44,7 +44,7 @@ export const NumberInputCell = <T extends RecordWithId>({
         ...TextInputProps?.InputProps,
       }}
       onChange={num => {
-        const newValue = num;
+        const newValue = num === undefined ? 0 : num;
```

Comment:
I would change to this - as it currently stands, you can clear the input and it gets set to `0`. then, you can press the down arrow to go lower than zero and the value changes to the min, which defaults to `1`. Setting the default to be the min feels more intuitive

```suggestion
        const newValue = num === undefined ? min : num;
```

that was along with 
line 17: `min = 1,` and move the comment from 53 to here as well
line 53: `min={min}`

Reply:
Yes, makes sense. I'll update this.